### PR TITLE
33 implement reefing driver

### PR DESCRIPTION
**Pull request by thomasmholder**

### CODE REVIEW

File: EosPayload/drivers/reefing_driver.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    self.set_reefing_level(50, logger)
+                elif self.latest_position.altitude > 4000:
+                    self.set_reefing_level(25, logger)
+                else:
+                    self.set_reefing_level(0, logger)
+            else:
+                self.set_reefing_level(0, logger)
+
+            # This check/update ensures that a crash somewhere that prevents position updates will cause a full dis-reef
+            self.old_position = self.latest_position
```
Comment (DarylDohner) [inline EosPayload/drivers/reefing_driver.py:67]: Is 30s without a position update a reasonable timeout period?

File: EosPayload/drivers/test_reefing_driver.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+class TestReefingDriver(ReefingDriver):
+    @staticmethod
+    def enabled() -> bool:
+        return False
+
+    @staticmethod
+    def get_device_id() -> Device:
+        return Device.MISC_TEST_3
```
Comment (DarylDohner) [inline EosPayload/drivers/test_reefing_driver.py:15]: This conflicts with test_queue_driver

File: EosPayload/drivers/mock_engineering_data_driver.py [hunk 3]
```diff
@@ -25,6 +25,7 @@ def get_device_id() -> Device:
         return Device.MISC_ENGINEERING_1
```
Comment (DarylDohner) [inline EosPayload/drivers/mock_engineering_data_driver.py:25]: Should mock engineering driver and the real engineering driver have the same device ID?

**Review by DarylDohner — CHANGES_REQUESTED** [review]
tiny things, see comments & ping me when ready for re-review

File: EosPayload/drivers/test_reefing_driver.py (hunk 2 above)
Comment (thomasmholder) [reply EosPayload/drivers/test_reefing_driver.py:15]: Fixed

File: EosPayload/drivers/reefing_driver.py (hunk 1 above)
Comment (thomasmholder) [reply EosPayload/drivers/reefing_driver.py:67]: I think so, even at twice our expected impact velocity it'll only fall 1000 feet, we're disreefing fully at 4000 feet, and lake Hartwell is at 660', so we have plenty of time. We can reduce it in the future.

**Review by DarylDohner — APPROVED** [review]
Approved provided you make a follow-up ticket to expand the device list and update these drivers

**Pull request merged**